xla/hlo/evaluator/hlo_evaluator.cc

```diff
@@ -377,8 +383,7 @@ std::optional<WhileCondComparisonOrNoOp> PatternMatchLoopCondComparison(
   if (!lhs.has_value() || !rhs.has_value()) {
     return std::nullopt;
   }
-  return WhileCondComparison{comparison->comparison_direction(),
-                             *std::move(lhs), *std::move(rhs)};
+  return WhileCondComparison{comparison->comparison_direction(), *lhs, *rhs};
```

Comment:
lhs and rhs are ParamIndexAndValue which is trivially copyable - removing std::move